### PR TITLE
restore rclone

### PR DESCRIPTION
**Pull request by dguibert**
###### Things done
- [X] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

By getting rid of go-packages.nix (#16017), rclone has removed (introduced by #15852).

### CODE REVIEW

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -14056,6 +14056,8 @@ in
     fftw = fftwSinglePrec;
   };
 
+  rclone = (callPackage ../applications/networking/sync/rclone { }).bin // { outputs = [ "bin" ]; };
```
Comment (kamilchm) [inline pkgs/top-level/all-packages.nix]: Go packages references in all-packages.nix was changed to default outputs after discussion https://github.com/NixOS/nixpkgs/pull/16129#discussion_r66704198

File: pkgs/applications/networking/sync/rclone/default.nix [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  goPackagePath = "github.com/ncw/rclone";
+
+  src = fetchgit {
+    inherit rev;
+    url = "https://github.com/ncw/rclone";
+    sha256 = "0np3k95zbhbaqnv251rz5j7nlpn8clbsnbiswmj55vxvsybk471x";
+  };
+
+  goDeps = ./deps.json;
+}
```
Comment (kamilchm) [inline pkgs/applications/networking/sync/rclone/default.nix]: meta?

**Comment by dguibert** [comment]
updated with meta and default outputs kept.

**Comment by joachifm** [comment]
Is this ready?

**Comment by Mic92** [comment]
Merged in 0ldmzcx5lxa81lcr39djcvyhd0ls11jlswj5877rinq3505ayf50

**Pull request closed without merging**